examples/fair_prep_results.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import os
+os.chdir("..")
+import shutil
+import warnings
+warnings.simplefilter('ignore')
+
+from fp.traindata_samplers import CompleteData
+from fp.missingvalue_handlers import CompleteCaseAnalysis
+from fp.dataset_experiments import GermanCreditDatasetSexExperiment
+from fp.dataset_experiments import RicciRaceExperiment
+from fp.scalers import NamedStandardScaler, NamedMinMaxScaler
+from fp.learners import NonTunedLogisticRegression, LogisticRegression, DecisionTree, NonTunedLogisticRegression, NonTunedDecisionTree          
+from fp.post_processors import NoPostProcessing,RejectOptionPostProcessing, EqualOddsPostProcessing, CalibratedEqualOddsPostProcessing
+from fp.pre_processors import NoPreProcessing, DIRemover, Reweighing
+
+import numpy as np
+get_ipython().run_line_magic('matplotlib', 'inline')
+import matplotlib.pyplot as plt
+
+seeds = [0xbeef, 0xcafe, 0xdead, 0xdeadcafe, 0xdeadbeef, 0xbeefcafe, 0xcafebeef, 50, 583, 5278, 100000, 0xefac,0xfeeb, 0xdaed, 0xefacdaed, 0xfeebdead]
+samplers = [CompleteData()]
+missings = [CompleteCaseAnalysis()]
+scalers = [NamedMinMaxScaler(), NamedStandardScaler()]
+learners = [NonTunedLogisticRegression(), LogisticRegression(), NonTunedDecisionTree(), DecisionTree()]
+post_processors = [NoPostProcessing(),RejectOptionPostProcessing(), CalibratedEqualOddsPostProcessing()]
+pre_processors = [NoPreProcessing(), DIRemover(1.0), DIRemover(0.5), Reweighing()]
+
+path = os.path.join(os.getcwd(), "logs")
+
+def extract_info(present_acc,present_di,present_fnr,present_fpr):
```

Comment:
please refactor this into a helper file, it should not be part of the notebook

Reply:
Done.